### DISCUSSION TITLE
/kick afk

### DISCUSSION THREAD

**Discussion by MrMcyeet** [Zrips/CMI#4284] [Feature Requests]
**Description of feature/improvement request:**
IMHO you should add a `/kick afk` that would kick everyone who is "Afk" according to CMI

---
### Detailed explanation of what it should do, when and in what situation: ###
When soneone with `cmi.command.afkkick` does `/afkkick` or `/kick afk` it would kick everyone who is "Afk"

### Basic explanation as to why this feature is needed and/or useful to have: ###
Useful for when the server is full, and you need to go through and kick afk players, freeing up space for normal people to come online.

**Comment by CloudeLecaw** [comment]
You can enable autoafk timer for kicking them automaticly if they are to long afk.
whats the benefit of having 56 players online on a 60 slot server when 50 of them stand afk...

A nice idea but mostly its not so good to allways add new commands.
Well, even if you need the afk kick why not have command variable for kick?
You could create a custom alias then for your wished /afkkick (executes cmi kick $1- -afk)

**so i recomment:**

> description: variable -afk for the kick command, if -afk set then playerName/allplayer that is/are afk get kicked

> permission: cmi.command.kick.afk
> command: /cmi kick [playerName/all] (message) (-afk) (-s)

**Comment by MrMcyeet** [comment]
> You can enable autoafk timer for kicking them automaticly if they are to long afk.
> whats the benefit of having 56 players online on a 60 slot server when 50 of them stand afk...
> 
> A nice idea but mostly its not so good to allways add new commands.
> Well, even if you need the afk kick why not have command variable for kick?
> You could create a custom alias then for your wished /afkkick (executes cmi kick $1- -afk)
> 
> **so i recomment:**
> 
> > description: variable -afk for the kick command, if -afk set then playerName/allplayer that is/are afk get kicked
> 
> > permission: cmi.command.kick.afk
> > command: /cmi kick [playerName/all] (message) (-afk) (-s)

+1 Great idea/improvement to my improvement
